### PR TITLE
chore(TYP-7505): Update Cypress version and run GH actions using latest node versions

### PR DESCRIPTION
**Pull request by mathio**

### CODE REVIEW

File: packages/embed/cypress.config.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  eyesIsDisabled: false,
+  eyesFailCypressOnDiff: true,
+  eyesDisableBrowserFetching: false,
+  eyesPort: 61632,
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
+    setupNodeEvents(on, config) {
+      return require('./cypress/plugins/index.js')(on, config)
+    },
```
Comment (antoniofull) [inline packages/embed/cypress.config.js:20]: Do we have to import the plugins?

File: packages/embed/cypress.config.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    specPattern: './e2e/spec/**/*.cy.{js,jsx,ts,tsx}',
```
Comment (antoniofull) [inline packages/embed/cypress.config.js:21]: Is this a new standard from the latest version of Cypress? Shall we change it in other repo's too?

Comment (mathio) [reply packages/embed/cypress.config.js:21]: This is the new standard coming from Cypress v10, it comes with a migration wizard that did all the changes related to Cypress in this PR.

**Review by scarciofolomarco — APPROVED** [review]

**Pull request merged**